test/dynamo/test_higher_order_ops.py

```diff
@@ -227,6 +227,26 @@ def f(x):
             f, (x,), ifdynstaticdefault(2, 3), expected_opcount=ifdynstaticdefault(1, 2)
         )
 
+    def test_wrap_pytree_args(self):
+        def f(x, y):
+            return wrap(lambda z: z[0].sin() * z[1].cos(), (x, y))
+
```

Comment:
Can you add a test for some nested python data structure? e.g. `{'x': x, 'y': (y, [z])}`